File: features/featureTwo/src/main/java/de/gfred/feature/two/di/FeatureTwoActivityBuilderModule.java
package de.gfred.feature.two.di;

import de.gfred.core.services.di.UserServiceModule;
import dagger.Module;
import dagger.android.ContributesAndroidInjector;
import de.gfred.feature.two.FeatureTwoActivity;

@Module
public abstract class FeatureTwoActivityBuilderModule
{
    @ContributesAndroidInjector(modules = {FeatureTwoActivityModule.class, UserServiceModule.class})
    public abstract FeatureTwoActivity contributeFeatureTwoActivity();
}

File: features/featureTwo/src/main/kotlin/de/gfred/feature/two/di/FeatureTwoActivityBuilderModule.kt
package de.gfred.feature.two.di

import dagger.Module
import dagger.android.ContributesAndroidInjector
import de.gfred.core.services.di.StringServiceModule
import de.gfred.feature.two.FeatureTwoActivity

@Module
abstract class FeatureTwoActivityBuilderModule {
    @ContributesAndroidInjector(modules = [FeatureTwoActivityModule::class, StringServiceModule::class])
    abstract fun contributeFeatureTwoActivity(): FeatureTwoActivity
}
